dev/ci/user-overlays/README.md

```diff
@@ -1,8 +1,22 @@
 # Add overlays for your pull requests in this directory
 
-An overlay is a file containing very simple logic to test whether we are currently building a specific pull request or git branch (useful so that overlays work on your own fork) and which changes some of the variables whose default can be found in [`ci-basic-overlay.sh`](/dev/ci/ci-basic-overlay.sh).
+When your pull request breaks an external development we test in our CI, you
+must prepare a patch (or ask someone to prepare a patch) to fix this development.
```

Comment:
"development" is not quite right.  Maybe "external code" or "external project", then at the end of the sentence "the" (or "that") instead of "this".  (Because "external" seems for something distant while "this" seems for something close IMHO.)